src/SendCloudApi.php

```diff
@@ -167,7 +167,9 @@ public function getUrl($url, $params = null) {
 		$api_url		= $this->apiUrl;
 		$api_parsed		= parse_url($api_url);
 		$resource_url	= parse_url($url);
-		$apiUrl = $api_parsed['scheme'].'://'.$this->getApiKey().':'.$this->getApiSecret().'@'.$api_parsed['host'].':'.$api_parsed['port'].'/';
+		// when port is url, add the port
```

Comment:
It seems like a word is missing from the comment.

// when port is url, add the port 

Should be something like 

// When the port is in the parsed url, add the port. 

Just like in your description on this PR. :-)